Libraries/StyleSheet/StyleSheet.js

```diff
@@ -13,7 +13,16 @@
 
 var StyleSheetRegistry = require('StyleSheetRegistry');
 var StyleSheetValidation = require('StyleSheetValidation');
-var flattenStyle = require('flattenStyle');
+
+/* TODO(brentvatne) docs are needed for this */
```

Comment:
Can you avoid adding comments like this inside the code. If those will be documented, they will be inside of those functions and this comment will unlikely be removed and be misleading.